diff -U 3 -dHrN a/install.sh b/install.sh
--- a/install.sh	2023-08-02 20:24:06.000000000 +0200
+++ b/install.sh	2023-08-02 20:35:00.274189167 +0200
@@ -44,7 +44,7 @@
 install -m 755 -d $destdir/share/ccdciel/data/resources
 install -m 755 -d $destdir/share/ccdciel/doc
 
-install -v -m 755 -s src/ccdciel  $destdir/bin/ccdciel
+install -v -m 755 src/ccdciel  $destdir/bin/ccdciel
 install -v -m 755 scripts/astrometry.sh  $destdir/share/ccdciel/scripts/astrometry.sh
 install -v -m 755 scripts/astrometry-online.sh  $destdir/share/ccdciel/scripts/astrometry-online.sh
 install -v -m 755 scripts/astrometry-online.bat  $destdir/share/ccdciel/scripts/astrometry-online.bat
