--- a/install.sh	2016-08-09 13:25:46.391953000 +0200
+++ b/install.sh	2016-08-16 18:30:13.364228770 +0200
@@ -27,7 +27,7 @@
 install -m 755 -d $destdir/share/ccdciel
 install -m 755 -d $destdir/share/ccdciel/scripts
 
-install -v -m 755 -s src/ccdciel  $destdir/bin/ccdciel
+install -v -m 755 src/ccdciel  $destdir/bin/ccdciel
 install -v -m 644 scripts/scope_park.script  $destdir/share/ccdciel/scripts/scope_park.script
 install -v -m 644 scripts/scope_unpark.script  $destdir/share/ccdciel/scripts/scope_unpark.script
 install -v -m 644 scripts/T_ccd_temp_down.script  $destdir/share/ccdciel/scripts/T_ccd_temp_down.script
