diff -U 3 -dHrN -- a/install.sh b/install.sh
--- a/install.sh	2018-03-08 19:17:54.000000000 +0100
+++ b/install.sh	2018-03-11 09:40:20.340203853 +0100
@@ -31,7 +31,7 @@
 install -m 755 -d $destdir/share/ccdciel/data/language
 install -m 755 -d $destdir/share/ccdciel/doc
 
-install -v -m 755 -s src/ccdciel  $destdir/bin/ccdciel
+install -v -m 755 src/ccdciel  $destdir/bin/ccdciel
 install -v -m 644 scripts/scope_park.script  $destdir/share/ccdciel/scripts/scope_park.script
 install -v -m 644 scripts/scope_unpark.script  $destdir/share/ccdciel/scripts/scope_unpark.script
 install -v -m 644 scripts/T_ccd_temp_down.script  $destdir/share/ccdciel/scripts/T_ccd_temp_down.script
